internal/dbus.go

```diff
@@ -98,9 +100,30 @@ func (n DBusNotify) Notify(
 
 	// Send Notification
 	nf := NewNotification()
-	nf.message = summary
+
 	parse_hints(&nf, hints)
 
+	// Setting min-width, left alignment
+	summary = fmt.Sprintf("%-60s", summary)
+	if nf.icon.value == nf.icon.NOICON {
+		summary += "  "
+	}
+	summary += "\u205F"
+
+	if body != "" {
+		nf.message = fmt.Sprintf("%s\n%s", summary, body)
+	} else {
+		nf.message = summary
+	}
+
+	// Using RegExp to add padding for all lines
+	nf.message = regexp.
+		MustCompile("^\\s*|(\n)\\s*(.)").
```

Comment:
nitpick: you could store this regex in a package-level variable to not recompile it every time this function is ran

Reply:
You're right, I'll do it!

Reply:
done! https://github.com/codelif/hyprnotify/pull/8/commits/024ba8b2f0fcc854c7a42cb8e84e05e9280997be